rules/carbonblack_rules/cb_audit_admin_grant.yml

```diff
@@ -0,0 +1,62 @@
+AnalysisType: rule
+RuleID: "CarbonBlack.Audit.Admin.Grant"
+LogTypes:
+    - CarbonBlack.Audit
+Description: 'Detects when a user is granted Admin or Super Admin permissions.'
+DisplayName: "Carbon Black Admin Role Granted"
+Enabled: true
+Filename: cb_audit_admin_grant.py
+Severity: High
+Tags:
+  - Privilege Escalation
+  - Account Manipulation
+Reports:
+  MITRE ATT&CK:
+    - TA0004:T1098
+Reference: https://docs.vmware.com/en/VMware-Carbon-Black-Cloud/services/carbon-black-cloud-user-guide/GUID-CF5ACD2C-A534-46C8-AE06-E1884DB37B58.html
+Threshold: 1
+DedupPeriodMinutes: 60
+Tests:
+    - Name: Super Admin granted
+      ExpectedResult: true
+      Log:
+        {
```

Comment:
Going forward I think we should stick with JSON logs; this is much more readable.